src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -348,6 +353,9 @@
   - name: --detach-acr
     type: string
     short-summary: Disable the 'acrpull' role assignment to the ACR specified by name or resource ID.
+  - name: --api-server-authorized-ip-ranges
+    type: str
+    short-summary: Comma seperated list of authorized apiserver IP ranges. Set to "" to disable. Set to 0.0.0.0/32 to allow all traffic in a previously restricted cluster.
```

Comment:
Set to 0.0.0.0/32 to disallow all traffic.

Reply:
Your right - chatted with Saurya, I misunderstood the difference between 0.0.0.0/32 and "". Here is how it works:
1. If you have IPWL enabled and you want to disable it then use "". When disabled then all traffic can access your api server (still protected by kubeconfig and RBAC of course)
2. If you want to restrict api server traffic to only your node pools then set IPWL to 0.0.0.0/32

Updating the help section now

Reply:
Updated